spec/spec_helper_acceptance.rb

```diff
@@ -84,9 +81,7 @@ def setup_puppet_on(host, opts = {})
   opts = {:agent => false, :mcollective => false}.merge(opts)
 
   step "Setup puppet on #{host}"
-  install_package host, 'puppet'
-  add_foss_defaults_on host
-  add_puppet_paths_on host
+  install_puppet_on host
```

Comment:
We should be adding options here for version and architecture for windows users?  
https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/spec/spec_helper_acceptance.rb#L7 as well as version numbers.  
Also the above `step` method should be removed as this has been yanked for beaker-rspec